### ISSUE TITLE
Cucumber step commandReturnShouldEqual should handle quoted strings

### ISSUE THREAD

**Issue by agajdosi** [minishift/minishift#1071]
Func commandReturnShouldEqual in integration_test.go should be updated to accept values in quotation marks for better readablity. 

Currently:
```godog
Then stdout should contain Default add-ons anyuid, admin-user, xpaas installed
```
Should be:
```godog
Then stdout should contain "Default add-ons anyuid, admin-user, xpaas installed"
```

**Comment by agajdosi** [comment]
Normally I would add it into some bigger issue, but this blocks me for downstream...

**Comment by hferentschik** [comment]
Merged via pull request #1072
